fw/cache.c

```diff
@@ -3145,11 +3181,16 @@ tfw_cache_start(void)
 	if (!(cache_cfg.cache || g_vhost->cache_purge))
 		return 0;
 
-	for_each_node_with_cpus(i) {
+	if (r = tfw_init_node_cpus())
```

Comment:
```suggestion
	if ((r = tfw_init_node_cpus()))
```
Did you try to compile your code?

Reply:
It looks like the last commit for some reasons haven't pushed.